ChaosMod/Effects/db/Player/PlayerZoomZoomCam.cpp

```diff
@@ -0,0 +1,72 @@
+/*
+    Effect by Moxi, based on code from DrUnderscore & ProfessorBiddle
+*/
+
+#include <stdafx.h>
+static Camera zoomCamera = 0;
+float camZoom = 80.f;
```

Comment:
these should be static. Otherwise this looks really good :)

Reply:
Not quite sur why it's needed, but done anyway!